recipe/meta.yaml

```diff
@@ -34,7 +34,7 @@ source:
     - patches/0008-link-grpc-_unsecure-to-grpc.patch
 
 build:
-  number: 0
+  number: 1
```

Comment:
Can you back out the build number bump please? It seems the migrator took precedence over the local cbc anyway, so updating the CBC doesn't actually change anything about the builds (though it's good to keep it up-to-date)

Reply:
I don't fully follow but I'll defer to your expertise here.